### PR TITLE
Create "farmOS 3.0.0" post

### PR DESCRIPTION
**Pull request by mstenta**
I wrote up a post to commemorate the release of farmOS 3.0.0. :-)

### CODE REVIEW

File: posts/2024/farmOS-3.0.0/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+- CSV importers for assets, logs, and taxonomy terms.
+- Asset ownership assignment.
+- Bulk actions for log categorization and assigning asset parents.
+- Lab test quantity type with "Test method" field, and "Tissue" lab test type.
+- Localization module for enabling translations.
+- Lots of small user interface/experience (UI/UX) features and improvements.
+
+And that doesn't include all the developer-facing features and improvements.
+For the full list, refer to the
+[farmOS Chanagelog](https://github.com/farmOS/farmOS/blob/3.x/CHANGELOG.md).
```
Comment (wotnak) [inline posts/2024/farmOS-3.0.0/index.md]:
```suggestion
[farmOS Changelog](https://github.com/farmOS/farmOS/blob/3.x/CHANGELOG.md).
```

Comment (mstenta) [reply posts/2024/farmOS-3.0.0/index.md]: Oops good eye @wotnak. I force pushed the commit with this fixed.

**Comment by wotnak** [comment]
The Open Collective images are wider than the content container, this is especially noticeable on mobile devices.
image

They should have `max-width:100%` style applied so they would fit into the content container. Limiting their width to the content container would also mean that they can be at most 655px wide, so the width argument in their urls could also be adjusted to not load an unnecessarily bigger images.

The best place to apply `max-width:100%`  would be probably in the global site styles, similarly as it was done for videos https://github.com/farmOS/farmOS.org/blob/main/src/theme.js#L37.  Images hosted directly on the farmOS website don't have this problem because Gatsby takes care of that for them automatically, but it doesn't process external images.

**Comment by mstenta** [comment]
> The Open Collective images are wider than the content container, this is especially noticeable on mobile devices.

Good call @wotnak. This is an existing issue in other pages as well:

- https://farmos.org/blog/2023/farmOS-2.0.0/ (I copied the code from this one)
- https://farmos.org/community/supporters/

Looks like it's working properly on this one:

- https://farmos.org/donate/

I will open a separate PR to fix that, and adjust this one in the same way...

**Comment by mstenta** [comment]
I wonder if we should just standardize on using the same embed code from the Donate page... in includes more than just people icons, but maybe not a bad thing. And it's super simple (and seems to be automatically responsive):

```
<script src="https://opencollective.com/farmOS/banner.js"></script>
```

**Comment by mstenta** [comment]
> The best place to apply `max-width:100%` would be probably in the global site styles

I tried a few things in the content itself, but I think you're right: we should apply a style in the global site styles to fix this.

Can we merge this and open follow-up PRs to fix the width of all instances together, as well as add site-wide styles?

**Review by wotnak — DISMISSED** [review]

**Review by paul121 — APPROVED** [review]

**Review by paul121 — CHANGES_REQUESTED** [review]
Actually maybe change the title to be consistent with "farmOS 2.0.0 is released" .... I like the shorter :-)

**Comment by mstenta** [comment]
> Actually maybe change the title to be consistent with "farmOS 2.0.0 is released" .... I like the shorter :-)

Hmm I don't feel too strongly either way, but worth noting: "has been released" is consistent with all our other announcements lately (forum, mastodon, opencollective). But maybe that's neither here nor there.

- https://farmos.discourse.group/t/farmos-3-0-0-has-been-released/1830
- https://fosstodon.org/@farmOS/111709175600342401
- https://opencollective.com/farmos/updates/farmos-3-0-0-has-been-released

Maybe also worth noting that PR #17 adds an old post for the farmOS 1.0 release, which is just titled "farmOS 1.0".

I agree we should start being more consistent. I prefer "has been released" myself and have been using that in other places. Maybe we can just settle on using that consistently from 3.0.0 onward? :-)

**Comment by mstenta** [comment]
Or we can start using "is released" consistently moving forward haha - I really don't mind either way. :laughing:

**Comment by paul121** [comment]
Ahh interesting. I guess I might even prefer the simple "farmOS 3.0.0" for the blog post titles.. Posting the new version number should imply it has been released? If we continue making blog posts for each major/minor release a complete sentence might be a little redundant in list of blog posts. A category with a badge for `Releases` could help identify these too.

But I do think "... has been released" as a complete sentence is the best especially for social media posts and maybe forum.

**Comment by mstenta** [comment]
I like that. I force pushed my commit (and updated this PR's title).

**Comment by mstenta** [comment]
> I might even prefer the simple "farmOS 3.0.0" for the blog post titles

We also do this for our commit messages, so I like it also for that reason. :-)

https://github.com/farmOS/farmOS/commit/f9225c05f4bb823f7738935735e75a961740efbe

**Review by paul121 — APPROVED** [review]

**Pull request merged**